packages/ws-worker/src/mock/sockets.ts

```diff
@@ -121,10 +121,15 @@ export class MockSocket {
       });
   }
 
-  channel(topic: string) {
+  channel(topic: string, params?: any) {
     if (!this.allChannels[topic]) {
       this.allChannels[topic] = mockChannel();
     }
+    // Store params on the channel for testing purposes
```

Comment:
It doesn't look like this is used, so I was going to say drop it

But actually! The tests you've generated jump through hoops to mock a  mock function :thinking: And it's wierd.

I'd much prefer this pattern - the mock channel simply saves its params, and the test can assert the value has been passed through